urllib3/util/ssl_.py

```diff
@@ -11,7 +11,6 @@
 
 SSLContext = None
 HAS_SNI = False
-create_default_context = None
```

Comment:
I have a strong distaste for changes that include irrelevant changes in them, but looking over the file before this change, we're not using the constant anywhere anyway.